examples/mongodbatlas_advanced_cluster/version-upgrade-with-pinned-fcv/README.md

```diff
@@ -0,0 +1,57 @@
+# MongoDB Atlas Provider -- Cluster with pinned FCV
+
+Example shows how to pin the FCV of a cluster making use of `pinned_fcv` block. This enables direct control to pin cluster’s FCV before performing an upgrade on the `mongo_db_major_version`. Users can then downgrade to the previous MongoDB version with minimal risk if desired, as the FCV is maintained.
+
+The unpin operation can be performed by removing the `pinned_fcv` block. **Note**: Once FCV is unpinned it will not be possible to downgrade the `mongo_db_major_version`. If FCV is unpinned past the expiration date the `pinned_fcv` attribute must be removed.
+
+The following [knowledge hub article](https://kb.corp.mongodb.com/article/000021785/) and [FCV documentation](https://www.mongodb.com/docs/atlas/tutorial/major-version-change/#manage-feature-compatibility--fcv--during-upgrades) can be referenced for more details.
+
+## Dependencies
+
+* Terraform MongoDB Atlas Provider v1.22.0
```

Comment:
will this be in 1.22.x or 1.23.x?

Reply:
yes this was outdated, will be included in 1.23.0